stripe/_stripe_object.py

```diff
@@ -111,13 +114,35 @@ def __init__(
         self._retrieve_params = params
         self._previous = None
 
-        object.__setattr__(self, "api_key", api_key)
-        object.__setattr__(self, "stripe_version", stripe_version)
-        object.__setattr__(self, "stripe_account", stripe_account)
```

Comment:
Just realized that because I've made these properties, they aren't settable in the same way as before. Looking into how to retain the previous setting behavior.

Reply:
https://github.com/stripe/stripe-python/pull/1201